<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_object_detect_model_select</class>
 <widget class="QWidget" name="widget_object_detect_model_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General params</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Confidence</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxConfidence">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="horizontalSliderConfidence">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>NMS</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxNMS">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSliderNMS">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Select model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxSelectModel"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxYoloV8">
        <property name="title">
         <string>yolo v8</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Process size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxProcessSizeYoloV8"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>222</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxConfidence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderConfidence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConfidence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxConfidence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxNMS</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderNMS</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderNMS</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxNMS</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
